src/main.rs

```diff
@@ -148,8 +148,16 @@ fn run(mut config: Config, options: cli::Options) -> Result<(), Box<Error>> {
     //
     // The monitor watches the config file for changes and reloads it. Pending
     // config changes are processed in the main loop.
-    let config_monitor = config.path()
-        .map(|path| config::Monitor::new(path, display.notifier()));
+    let mut config_monitor = None;
```

Comment:
I've got a few nits about this section of code:

1. `config_monitor` is getting wrapped in a second `Option` and ends up with the type `Option<Option<config::Monitor>>`.
2. Code to create the config monitor is not dry (`Monitor::new` called identically in two places).
3. Kind of hard to follow the logic.

I would propose rewriting this as a match expression. That is, something like:

```rust
let config_monitor = match (opts.live_reload, config.live_reload) {
    // Start monitor if CLI flag says yes
    (Some(true), _) |
    // Or if no CLI flag was passed and the config says yes
    (None, true) => (), // create the monitor here
    // Otherwise, don't start the monitor
    _ => None,
};
```